README.md

```diff
@@ -470,7 +470,8 @@ always be made under a new AST version, while previous formats continue to be su
 time.
 
 Old AST versions may be deprecated and subsequently removed. However, AST versions will only be
-removed in conjunction with a version increase of the AST extension from 0.x to 0.(x+1).
+removed in conjunction with a version increase of the AST extension from x.z to (x+1).0.
+Deprecations should be performed in minor releases instead of patch releases.
```

Comment:
I would rewrite this whole paragraph as "Old AST versions may be deprecated in minor versions and removed in major versions of the AST extension".